### ISSUE TITLE
Exception using cuDNN 6.0

### ISSUE THREAD

**Issue by lpaatero** [deeplearning4j/deeplearning4j#3387]
Test code (in gist) works without cuDNN using current 8.1 snapshot and with 8.0 release with or without cuDNN 5.0.
When run using cuDNN 6.0, produces unexpected exception in graph fit method.

```
Exception in thread "main" java.lang.RuntimeException: cuDNN status = 3
	at org.deeplearning4j.nn.layers.convolution.CudnnConvolutionHelper.checkCudnn(CudnnConvolutionHelper.java:60)
	at org.deeplearning4j.nn.layers.convolution.CudnnConvolutionHelper.backpropGradient(CudnnConvolutionHelper.java:274)
	at org.deeplearning4j.nn.layers.convolution.ConvolutionLayer.backpropGradient(ConvolutionLayer.java:160)
	at org.deeplearning4j.nn.graph.vertex.impl.LayerVertex.doBackward(LayerVertex.java:123)
	at org.deeplearning4j.nn.graph.ComputationGraph.calcBackpropGradients(ComputationGraph.java:1518)
	at org.deeplearning4j.nn.graph.ComputationGraph.computeGradientAndScore(ComputationGraph.java:1183)
	at org.deeplearning4j.optimize.solvers.BaseOptimizer.gradientAndScore(BaseOptimizer.java:160)
	at org.deeplearning4j.optimize.solvers.StochasticGradientDescent.optimize(StochasticGradientDescent.java:59)
	at org.deeplearning4j.optimize.Solver.optimize(Solver.java:57)
	at org.deeplearning4j.nn.graph.ComputationGraph.fit(ComputationGraph.java:1038)
	at org.deeplearning4j.nn.graph.ComputationGraph.fit(ComputationGraph.java:997)
	at org.deeplearning4j.nn.graph.ComputationGraph.fit(ComputationGraph.java:758)
	at net.gowrite.dcnn.Testcudcnn.main(Testcudcnn.java:127)

```
Full test code:
https://gist.github.com/lpaatero/ff133a3ea9fc365eebda5e6c0df0365c

Running on OpenSuse 42.2. using nvidia 1070 and cuda 8.0.

[Testcudcnn.zip](https://github.com/deeplearning4j/deeplearning4j/files/989270/Testcudcnn.zip)

**Comment by lpaatero** [comment]
Problem resolved; seems to have been some kind of build problem in my end. Still not quite sure what.

**Comment by saudet** [comment]
Ah yes, forgot to comment here. It's probably been fixed with pull #3388 that I merged yesterday.
